### PR TITLE
Adds assets and changes the main design

### PR DESCRIPTION
**Pull request by hjJunior**
Just adapting design 

## Before
captura de tela 2019-02-21 as 13 44 24

## Now
captura de tela 2019-02-21 as 13 26 11
captura de tela 2019-02-21 as 13 26 26

### CODE REVIEW

File: assets/stylesheets/game.css [hunk 1]
```diff
@@ -1,10 +1,78 @@
...
-    background: #261C2A;
-    width: 50%;
+    width: 800px;
+    height: 750px;
     margin-left: auto;
     margin-right: auto;
+    background-image: url('../images/floor.jpg');
+    background-size: 10rem;
+
+  	-webkit-touch-callout: none;
```
Comment (Kutomore) [inline assets/stylesheets/game.css]: shouldn't this be aligned with the rest?

File: assets/stylesheets/game.css [hunk 2]
```diff
@@ -1,10 +1,78 @@
...
+	background: -moz-linear-gradient(top, #ffffff 0%, #f1f1f1 50%, #e1e1e1 51%, #f6f6f6 100%);
+	background: -webkit-linear-gradient(top, #ffffff 0%, #f1f1f1 50%, #e1e1e1 51%, #f6f6f6 100%);
+	background: linear-gradient(to bottom, #ffffff 0%, #f1f1f1 50%, #e1e1e1 51%, #f6f6f6 100%);
+	filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ffffff', endColorstr='#f6f6f6', GradientType=0);
+
+	display: flex;
+	height: 4rem;
+	justify-content: space-between;
+    padding-right: 1rem;
+    padding-left: 1rem;
```
Comment (Kutomore) [inline assets/stylesheets/game.css]: spacing again

**Comment by hjJunior** [comment]
I think something wrong with my idea, I'll fix the indentation issues thx

File: assets/stylesheets/game.css [hunk 3]
```diff
@@ -1,10 +1,78 @@
...
+	font-size: 19px;
+}
+
+.wall {
+	height: 80px;
+	background-image: url('../images/wall.jpg');
 }
 
 .character {
     height: 50px;
```
Comment (Kutomore) [inline assets/stylesheets/game.css]: Identation

File: assets/stylesheets/game.css [hunk 4]
```diff
@@ -1,10 +1,78 @@
+@font-face {
+  font-family: arcadeclassic;
```
Comment (Kutomore) [inline assets/stylesheets/game.css]: same here

File: index.html [hunk 5]
```diff
@@ -6,15 +6,29 @@
 	</head>
 	<body>
 		<div id='game'>
+			<div class="stats">
+			    <div class="stats__money">
```
Comment (Kutomore) [inline index.html]: spaces instead of tab

File: index.html [hunk 6]
```diff
@@ -6,15 +6,29 @@
...
+					<img src="assets/images/money.png" height="30">
+			    	<div id="money"></div>
```
Comment (Kutomore) [inline index.html]: spaces instead of tab

File: index.html [hunk 7]
```diff
@@ -6,15 +6,29 @@
...
+				</div>
+			    <div class="project"></div>
```
Comment (Kutomore) [inline index.html]: spaces instead of tab

File: assets/javascript/game.js [hunk 8]
```diff
@@ -35,14 +35,14 @@ function click(){
 }
 
 function hireHelp() {
-    if(money > (Math.pow(helper_count, 2)+1)){
+    if(money > (Math.pow(helper_count, 2) + 1)) {
```
Comment (Kutomore) [inline assets/javascript/game.js]: spaces instead of tab

File: assets/javascript/game.js [hunk 9]
```diff
@@ -35,14 +35,14 @@ function click(){
...
+        helper_count++;
         addNewHelper();
     }
 }
 
 function addNewHelper() {
-    office.innerHTML += '<img class="character" src="assets/images/main_guy.png">'
-}
\ No newline at end of file
+    employers.innerHTML += '<img class="character" src="assets/images/main_guy.png">'
```
Comment (Kutomore) [inline assets/javascript/game.js]: spaces instead of tab

File: assets/javascript/game.js [hunk 10]
```diff
@@ -1,7 +1,7 @@
 var money = 0;
 var click_value = 1;
 var helper_count = 0;
-const office = document.getElementById('office');
+const employers = document.querySelector('#office > .employers')
 
 window.onload = function startGame(){
     document.getElementById('office').addEventListener("click", click, false);
```
Comment (Kutomore) [inline assets/javascript/game.js]: spaces instead of tab

File: assets/javascript/game.js [hunk 11]
```diff
@@ -18,7 +18,7 @@ function outsourcedMoney() {
 }
 
 function updateMoney() {
-    document.getElementById('money').innerHTML= "You have: $"+money
+    document.getElementById('money').innerHTML = money
```
Comment (Kutomore) [inline assets/javascript/game.js]: spaces instead of tab

**Review by Kutomore — APPROVED** [review]

**Pull request merged**